Bottom: 300165e6eb541db81cdb4053ae56d5e327b4a009
Top:    6a0ebb480816ac35d07b46efb7b6739bbfa30628
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-04-22 18:28:29 -0400

Refresh of intc.patch

---

diff --git a/drivers/i2c/busses/i2c-pnx.c b/drivers/i2c/busses/i2c-pnx.c
index ffeaceb..855c534 100644
--- a/drivers/i2c/busses/i2c-pnx.c
+++ b/drivers/i2c/busses/i2c-pnx.c
@@ -718,7 +718,6 @@ static int __devinit i2c_pnx_probe(struct platform_device *pdev)
 		dev_err(&pdev->dev, "Failed to get IRQ from platform resource\n");
 		goto out_irq;
 	}
-	printk("i2c_pnx_probe irq is %d\n", alg_data->irq);
 	ret = request_irq(alg_data->irq, i2c_pnx_interrupt,
 			0, pdev->name, alg_data);
 	if (ret)
